primitives/src/constants.rs

```diff
@@ -84,6 +84,9 @@ pub mod chain {
 
 	/// We allow for
 	pub const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND / 2;
+
+	pub const GALACTIC_COUNCIL_ACCOUNT: [u8; 32] =
```

Comment:
do we really need it here? can we move it somewhere lower? we cannot use this account on testnet/locally for example

Reply:
moved to basilisk runtime